modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Advantages and disadvantages of non-versioned and versioned cluster tasks
+
+.Advantages and disadvantages of non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| Cluster task variant | Advantages | Disadvantages
+
+| Non-versioned cluster task 
+a| 
+* If you prefer {pipelines-shortname} with latest updates and bug fixes, use the non-versioned cluster task variant. 
```

Comment:
`{pipelines-shortname}` what does this resolve to when the doc is rendered.

I'm asuming it is "pipelines" as in an instance of the CRD `Pipeline` (what customer use to setup their workflows). 🙂

Reply:
```suggestion
* If you prefer {pipelines-shortname} with the latest updates and bug fixes, use the non-versioned cluster task variant. 
```